plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -105,13 +109,14 @@ public PluginManager(Config cfg) {
      * the failed plugins
      */
     public void start() {
-        if (!refDir.exists()) {
+        if (refDir.exists()) {
             try {
-                Files.createDirectories(refDir.toPath());
+                FileUtils.deleteDirectory(refDir);
```

Comment:
What if there are already plugins in the ref dir? Some people copy plugins into the refDir and then run the install plugins to add additional ones from the update center. This would not allow that usage case.

Reply:
Do it after the install plugins runs?

Reply:
Its either that or we need to download plugins somewhere else and then move them when completed, but this is definitely the easiest fix on our side

Reply:
FTR https://github.com/jenkinsci/plugin-installation-manager-tool/issues/173